pkg/oc/bootstrap/docker/openshift/rbac.go

```diff
@@ -1,53 +0,0 @@
-package openshift
-
-import (
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/pkg/apis/rbac"
-
-	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
-)
-
-// Roles
-const (
-	ClusterServiceBrokerAdminRoleName = "system:openshift:clusterservicebroker-client"
-)
-
-// GetServiceCatalogRBACDelta returns a cluster role with the required rules to bootstrap service catalog
-func GetServiceCatalogRBACDelta() []rbac.ClusterRole {
-	return []rbac.ClusterRole{
-		{
-			ObjectMeta: v1.ObjectMeta{
-				Name: bootstrappolicy.AdminRoleName,
```

Comment:
This PR should be a 6 line change delta where you add the appropriate label to each cluster role and give each cluster role a unique name.  For example:

```
{
	ObjectMeta: v1.ObjectMeta{
		Name: "system:openshift:service-catalog:aggregate-to-admin",
		Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-admin": "true"},
},
...
```